metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -87,12 +87,14 @@ def echo_none(*args, **kwargs):
         except err as e:  # noqa F841
             pass
 
-@batch.command(help="List running Batch tasks of this flow")
-@click.option(
-    "--my-runs", default=False, is_flag=True, help="Run the command over all tasks."
-)
-@click.option("--user", default=None, help="List tasks for the given user.")
-@click.option("--run-id", default=None, help="List tasks corresponding to the run id.")
+
```

Comment:
I am the first to fall in the trap but formatting changes are difficult to parse in parallel with actual code change :(.

Reply:
Apologies. There are a few, mostly related to comments and help messages to maintain consistency.